lib/cs_guide/accounts/accounts.ex

```diff
@@ -0,0 +1,104 @@
+defmodule CsGuide.Accounts do
```

Comment:
is this _generic_ Account? or is it required for registering a venue?

Reply:
This is just the context for the different account functions we might have. We're not currently using it, but it's created as part of the Phoenix generator, and will have some content in the future

Reply:
@Danwhy thanks for clarifying. we can update as required laters. good for now. 👍